clone/step_clone.go

```diff
@@ -10,11 +10,12 @@ import (
 )
 
 type CloneConfig struct {
-	Template    string `mapstructure:"template"`
-	DiskSize    int64  `mapstructure:"disk_size"`
-	LinkedClone bool   `mapstructure:"linked_clone"`
-	Network     string `mapstructure:"network"`
-	Notes       string `mapstructure:"notes"`
+	DiskSize      int64    `mapstructure:"disk_size"`
```

Comment:
It's hard to track difference if parameters are rearranged

Reply:
True. I will undo the rearrangements for clarity.